desk/app/groups.hoon

```diff
@@ -678,6 +685,9 @@
   ++  go-update
     |=  [=time =diff:g]
     ^+  go-core
+    ?:  &(!=(our.bowl p.flag) from-self) 
+      =.  cor  (emit (update:go-pass [flag [time diff]]))
+      go-core
```

Comment:
this logic need to go in the `%group-action` case of the +poke arm, see the pattern established in `%chat`